src/Api/Service/MyParcelApiService.php

```diff
@@ -67,20 +68,16 @@ public function getHeaders(): array
      */
     protected function getUserAgentHeader(): string
     {
-        $userAgentStrings = [];
-        $userAgents       = array_merge(
-            [$this->userAgent],
-            ['MyParcel-PDK' => InstalledVersions::getPrettyVersion(self::PACKAGE_NAME)],
-            ['php' => PHP_VERSION]
-        );
-
-        foreach ($userAgents as $key => $value) {
-            if (is_int($key)) {
-                $userAgentStrings[] = $value;
-                continue;
-            }
+        $userAgentValue   = array_values($this->userAgent);
+        $userAgentStrings = [
+            array_key_first($this->userAgent) => array_shift($userAgentValue),
```

Comment:
`array_key_first` is een php 7.3 functie. Plus, dit kan allemaal 10x makkelijker en veiliger, je bent echt heel moeilijk aan het doen

```php
$userAgentStrings = [];
$userAgents       = [
    'MyParcel-PDK' => InstalledVersions::getPrettyVersion(Pdk::PACKAGE_NAME),
    'php'          => PHP_VERSION,
] + $this->userAgent;

foreach ($userAgents as $platform => $version) {
    $userAgentStrings[] = sprintf('%s/%s', $platform, $version);
}

return implode(' ', $userAgentStrings);
```